packages/beeq/src/components/notification/bq-notification.tsx

```diff
@@ -231,8 +234,12 @@ export class BqNotification {
   // ===================================
 
   render() {
+    const style = {
+      ...(this.border && { '--bq-notification--border-radius': `var(--bq-radius--${this.border})` }),
+    };
```

Comment:
Please leave an empty line between the const declaration and the return statement.